go/ct/rlz/parameters.go

```diff
@@ -154,6 +154,9 @@ var gasParameterSamples = []U256{
 	NewU256(1),
 	NewU256(1 << 10),
 	NewU256(st.MaxGasUsedByCt),
+	NewU256(math.MaxInt64),
+	NewU256(math.MaxInt64 + 1),
+	NewU256(math.MaxUint64),
```

Comment:
Did this change detect the issue fixed in this PR?

Reply:
yes, I first added this to the PR, then it failed for lfvm, then I fixed it the lfvm interpreter and the CT calls worked again.